qmlbind/include/qmlbind/qmlbind_global.h

```diff
@@ -51,9 +51,8 @@ typedef QJSValueIterator qmlbind_iterator;
 typedef QByteArray qmlbind_string;
 
 // Use shared pointer for widely referenced classes
-typedef std::shared_ptr<QmlBind::Interface> qmlbind_interface;
 typedef std::shared_ptr<QmlBind::MetaObject> qmlbind_metaobject;
-typedef std::shared_ptr<QmlBind::Exporter> qmlbind_exporter;
+typedef std::unique_ptr<QmlBind::Exporter> qmlbind_exporter;
```

Comment:
How about using raw pointer instead of `std::unique_ptr` in C layer? (you have to delete exporters manually in C anyway and maybe it slightly improves performance)